sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosClientEncryptionKeyProperties.java

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.ClientEncryptionKey;
+import com.azure.cosmos.implementation.Resource;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Details of an encryption key for use with the Azure Cosmos DB service.
+ */
+public class CosmosClientEncryptionKeyProperties {
```

Comment:
any class in `com.azure.cosmos.models` is public either we should move this to encryption project or we should annotate all the methods with beta api.

Reply:
Annotated all the apis